src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -3010,9 +3052,24 @@ namespace winrt::TerminalApp::implementation
             co_return;
         }
 
+        auto pkgList = co_await _FindPackageAsync(args.MissingCommand());
+        for (int retries = 0; !pkgList && retries < 3; ++retries)
+        {
+            pkgList = co_await _FindPackageAsync(args.MissingCommand());
+        }
+
+        // no packages were found, nothing to suggest
+        if (pkgList.Size() == 0)
```

Comment:
```suggestion
        if (!pkgList || pkgList.Size() == 0)
```
In case you hit the retry limit and still have a nullptr.